source/DoorModel.cpp

```diff
@@ -0,0 +1,9 @@
+﻿#include "DoorModel.h"
+
+/**
+ * Disposes all resources and assets of this door.
+ *
+ * Any assets owned by this object will be immediately released.  Once
+ * disposed, a door may not be used until it is initialized again.
+ */
+void DoorModel::dispose() {}
```

Comment:
Otherwise it's an empty cpp lol. I can remove if you'd prefer that

Reply:
I was just wondering why you made a `.cpp` if all the methods were inlined in the `.h` file. Since you've already made it and added it to both projects, go ahead and move this method to the header file and leave the blank cpp. We can populate it later.